modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/utils/utils.es.js

```diff
@@ -122,15 +122,26 @@ export function normalize(ratingValue) {
 }
 
 export function stringToSlug(text) {
+	const hyphens = /-+/g;
 	const whiteSpaces = /\s+/g;
 
-	return text.replace(whiteSpaces, '-').toLowerCase();
+	text = text.toLowerCase();
+	text = text.replace(hyphens, '--');
+
```

Comment:
I've just realized that this is already ordered alphabetically... (hypens and whiteSpaces). So maybe Brian was referring to add an breakline between the toLowerCase method and the first replace method? I think that you were right when we talked about it and the breakline is needed between this to lines